avocado/utils/disk.py

```diff
@@ -154,3 +154,20 @@ def get_filesystem_type(mount_point='/'):
             _, fs_file, fs_vfstype, _, _, _ = mount_line.split()
             if fs_file == mount_point:
                 return fs_vfstype
+
+
+def is_root_device(device):
+    """
+    check for root disk
+
+    :param device: device to check
+    :returns: True or False, True if given device is root disk
+              otherwise will return False.
+    """
+    cmd = "lsblk --j -o MOUNTPOINT,PKNAME"
+    output = process.run(cmd)
+    result = json.loads(output.stdout_text)
+    for item in result['blockdevices']:
+        if item['mountpoint'] == "/" and str(item['pkname']) in device:
```

Comment:
The way that I'm reading the docstrings here, is that `device` is a string, not a list. Also, the `pkname` value on the output command is also a string. So we need either change the docstring to accept "devices" and make clear that is list, or change this `in` by a `==`.

Reply:
Indeed, my original proposal was:

`if item['mountpoint'] == "/" and device == str(item['pkname']):`

I thought the diff was applied here and I did not check the condition before approve.

IMHO, it does not make sense to accept a list here as argument, but a string, so the check should be changed to `==`.

Reply:
@willianrampazzo  @beraldoleal actually i put in just because, while we passing the device it is like "/dev/sda" and here we are getting output of str(item['pkname']) is 'sda'. to avoid extra lines i keep it like this, "str(item['pkname']) in device".   ok i will change that.